### PR TITLE
Documentation of reconciliation flow and reports

### PR DESCRIPTION
**Pull request by rominariedel**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -2104,6 +2104,38 @@ Notifies API that a reconciliation file was uploaded, so that it can be processe
...
+Returns the reports of the current client.
+
+### reports [GET]
+
++ Request (application/json)
+
+    + Header
+
+            Accept: application/json
+            Authorization: eyJ0eXAiOiJKV1QiLCJhbiAoOiJIUzI1NiJ9.eyJvX2F1dGhfYXBwbGljYXRpb25faWQiOjF9.h7ZNPpbd_Bu8yL27fmCAI_wgyZJI44b5eRRxwWUcwaQ
```
Comment (aguspagnoni) [inline apiary.apib]: only being authorized is needed to do a GET to this controller? doesn't it have a `:page` parameter? or it responds all reports available?

Comment (rominariedel) [inline apiary.apib]: Nope, it doesn't receive any `:page` parameter or anything. Just the authorization! It responds with all the reports available but as I see in front, it only shows like the last 6. We should consider changing that it the future..

Comment (aguspagnoni) [inline apiary.apib]: ook, is it a front card or an api? or both? could you do me the favor of creating it under 'Mejora'? thaanks :)

**Comment by aguspagnoni** [comment]
@rominariedel another thing, I will merge this, afterwards you'll have to write the release inside `api-docs`  repo, with a brief summary

File: apiary.apib (hunk 1 above)
Comment (rominariedel) [inline apiary.apib]: I think both. I'd ask @mdoublier if the client needs to see old reports or not

**Pull request merged**